clippy_lints/src/methods/mod.rs

```diff
@@ -4284,6 +4285,29 @@ declare_clippy_lint! {
     "map of a trivial closure (not dependent on parameter) over a range"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    ///
+    /// Checks for `repeat().take()` that can be replaced with `repeat_n()`.
+    ///
+    /// ### Why is this bad?
+    ///
+    /// Using `repeat_n()` is more concise and clearer.
+    ///
+    /// ### Example
+    /// ```no_run
+    /// let _ = std::iter::repeat(10).take(3);
+    /// ```
+    /// Use instead:
+    /// ```no_run
+    /// let _ = std::iter::repeat_n(10, 3);
+    /// ```
+    #[clippy::version = "1.85.0"]
+    pub REDUNDANT_REPEAT_TAKE,
```

Comment:
What do you think about `manual_repeat_n`? "redundant" makes this sound like the whole operation can be removed, but this is rather a different way to write `repeat_n()`

Reply:
Okay, so how do you think this lint should be renamed? Would simply naming it `repeat_take` be a good lint name? (Sorry for my lack of understanding of the nuances of English vocabulary)

Reply:
`manual_repeat_n` would be my suggestion (the comment mentioned this ^^)

Reply:
Oh I misread your message, then I'll rename this with `manual_repeat_n`, thank you.